src/index.js

```diff
@@ -431,7 +439,9 @@ export default class HyperScreen extends React.Component {
           targetElement = currentElement;
         }
 
-        newRoot = Behaviors.performUpdate(action, targetElement, newElement);
+        if (newElement) {
+          newRoot = Behaviors.performUpdate(action, targetElement, newElement);
+        }
```

Comment:
this is an unexpected change with regards to the feature this PR is introducing, is this solving a bug?

Reply:
Yes, we are returning [null](https://github.com/Instawork/hyperview/blob/sree/handle-fragment-network-error/src/index.js#L324) from `fetchElement` when there is an error, so this tries to perform update on null and breaks.